cypress.config.js

```diff
@@ -10,10 +10,17 @@ module.exports = defineConfig({
           const email = faker.internet.email();
           const randomNumber = Math.floor(Math.random(1000) * 1000);
           return {
-            username: faker.name.firstName() + randomNumber,
+            username: faker.person.firstName() + randomNumber,
             email: email.toLowerCase(),
             password: '12345Qwert!'
```

Comment:
Using a static password for all generated users might not be the best approach. Consider generating a random password for each user for better security. You can use `faker.internet.password()` for this purpose.